### DISCUSSION TITLE
有没有可能支持邀请某人进群的功能

### DISCUSSION THREAD

**Discussion by fakedestinyck** [Mrs4s/go-cqhttp#790] [Ideas]
请问一下不知道能不能添加一个 邀请某人进群的API呢？

**Comment by Ink-33** [comment]
无可能![IMG_20210404_120353.jpg](https://user-images.githubusercontent.com/51873347/113498251-dc824280-953d-11eb-874f-0eedf0a32f19.jpg)

**Reply by fakedestinyck** [reply]
谢谢 就是没找到截图的文档在哪。。。

**Reply by Ink-33** [reply]
> 谢谢 就是没找到截图的文档在哪。。。

Mrs4s/MiraiGo
